common/app/views/fragments/discussionFooter.scala.html

```diff
@@ -104,7 +99,15 @@ <h2 class="container__meta__title">
                     </div>
 
                     <div class="modern-visible">
-                        @sectionHeading
+
+                        <div class="discussion__heading">
+                            <div class="container__meta">
+                                @if(discussionClosed) {
+                                    <p class="container__meta__item">
+                                    This discussion is closed for comments.
+                                </p>}
+                            </div>
+                        </div>
```

Comment:
Keeping this message as the conditional is Scala code and I think it's useful information for the user. If a discussion is closed for comments, the message will show regardless of whether the user is signed in or not.